src/SW_Weather.c

```diff
@@ -945,7 +1138,12 @@ void SW_WTH_new_day(void) {
     /* get the daily weather from allHist */
     if (
       missing(w->allHist[yearIndex]->temp_avg[day]) ||
-      missing(w->allHist[yearIndex]->ppt[day])
+      missing(w->allHist[yearIndex]->ppt[day]) ||
+      missing(w->allHist[yearIndex]->cloudcov_daily[day]) ||
+      missing(w->allHist[yearIndex]->windspeed_daily[day]) ||
+      missing(w->allHist[yearIndex]->r_humidity_daily[day]) ||
+      (missing(w->allHist[yearIndex]->shortWaveRad[day]) && w->has_rsds) ||

```

Comment:
Could you please add a comment in the code why we treat radiation here differently, i.e., `(missing(w->allHist[yearIndex]->shortWaveRad[day]) && w->has_rsds)`?

We will add the special treatment for cloud cover with the next phase?